_docs/Contrib.md

```diff
@@ -1,33 +1,62 @@
+        * User supplied attributes - are placed on a component by the developer who uses it in an application. All supported attributes need to be explicitly added in the props interface of the component (as optional).
 
-    The current relevant attributes for all components are the text content attributes, used in interactive components, components with landmark roles and components with widget roles (see [explanation](https://www.paciellogroup.com/blog/2017/07/short-note-on-aria-label-aria-labelledby-and-aria-describedby/)). The attributes are aria-label, aria-labelledby, and aria-describedby.
+            The current relevant attributes for all components are the text content attributes, used in interactive components, components with landmark roles and components with widget roles (see [explanation](https://www.paciellogroup.com/blog/2017/07/short-note-on-aria-label-aria-labelledby-and-aria-describedby/)). The attributes are aria-label, aria-labelledby, and aria-describedby. See example in next [section](#sample-implmentation).
 
-* Internal Implementation Attributes - are placed by the component code either on the root or on the relevant children in a composite component. Examples for these attributes are aria-autocomplete, aria-current, aria-checked and so on. The relevant attributes for a component will be added in the spec by the product manager.
+        * Internal Implementation Attributes - are placed by the component code either on the root or on the relevant children in a composite component. Examples for these attributes are aria-autocomplete, aria-current, aria-checked and so on. The relevant attributes for a component will be added in the spec by the product manager. See example in next [section](#sample-implmentation).
 
-### Implementing Accessibility
+    3) **Implementing keyboard support**: a set of keyboard behaviors implemented for key presses done when the component is in focus. For each component type, the w3 organization has provided clear instructions on how to correctly implement them. For components not in the draft, the Wix accessibility team will provide specific instructions.
+
+        Check out the accordion [example](https://www.w3.org/TR/wai-aria-practices/examples/accordion/accordion.html) in the draft.
+
+    4) **Focus**: in order for an interactive component to be supported by assistive technologies, its ability to gain focus is crucial. It is important for a composite component (i.e. dropdown, radio group) to gain focus through tab and to explicitly set the tabindex on its children. See [Keyboard navigation inside components](https://www.w3.org/TR/wai-aria-practices/#kbd_general_within) for an explanation on the correct use of tabindex in composite components.
+
+    The following shows the DOM representation of a custom Radio Group, which when gaining focus through tab, assigns the focus to the checked radio button:
```

Comment:
missing explanation: what in the code causes that behavior? is it in inherent property of `radiogroup`, or something else?